test/e2e/store_offset.test.ts

```diff
@@ -30,10 +30,14 @@ describe("declare consumer", () => {
       (message: Message) => messages.push(message.content)
     )
 
-    await consumer.storeOffset(Offset.offset(1n).value!)
+    await consumer.storeOffset(1n)
```

Comment:
This test should be changed:
1. remove the skip
2. store the offset while consuming
3. Verify with the query we have stored everything